frontend/src/Pages/hamburger.jsx

```diff
@@ -155,6 +157,10 @@ const Hamburger = () => {
   const helpClick = () => {
     navigate("/Help");
   };
+  const FaqClick = () => {
+    navigate("/FAQ");
+  };
+
```

Comment:
fix the dom link and css it is not opening and overlapping 

![image](https://github.com/user-attachments/assets/96dfa357-6c6e-436f-8d85-d1afd714da31)